### ISSUE TITLE
Add a FAQ section

### ISSUE THREAD

**Issue by rhuss** [knative/client#651]
## Describe the feature:

A FAQ should shortly describe how to realize common use cases with a single command. Actually we could borrow some ideas of https://knative.tips and show how those tips could be implemented via `kn` (as it looks like that the ghost behind knative.tips is not very positive to include `kn` on that web site). But other FAQs are equally well suited. Ideally, each FAQ shows a single kn illustrating a certain use case.

This is part of #510

**Comment by maximilien** [comment]
Happy to contribute to this @rhuss. Did you have a list of initial FAQ in mind? If not I can suggest a few and we can add them here as tasks. Cool?

**Comment by rhuss** [comment]
Good idea,  lets collect some FAQs here until we decide where to put them eventually.

**Comment by rhuss** [comment]
I think a dedicated document (or multiple documents) below https://github.com/knative/docs/client would be the proper place for this.
